topic/timeseries/time-series-decomposition.ipynb

```diff
@@ -106,12 +106,37 @@
     "engine = sa.create_engine(CONNECTION_STRING, echo=os.environ.get('DEBUG'))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "First, import data into CrateDB. This is a shorthand notation for the same code\n",
+    "illustrated in `timeseries-queries-and-visualization.ipynb`, running corresponding\n",
+    "SQL DDL and DML statements, to load the data."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "from cratedb_toolkit.datasets import load_dataset\n",
+    "\n",
+    "dataset = load_dataset(\"tutorial/weather-basic\")\n",
+    "dataset.dbtable(dburi=CONNECTION_STRING, table=\"weather_data\").load()"
```

Comment:
That significantly reduces the boilerplate code mentioned on a different comment, building upon improvements from other patches.

Because the default setting for the optional keyword argument `if_exists` is `noop`, it will also save significant resources when running the same code over and over again, because data will only be imported once, which is even more convenient than pandas' `append` or `replace` variants on its I/O methods.

- https://github.com/crate/cratedb-datasets
- https://github.com/crate-workbench/cratedb-toolkit/issues/89
- https://github.com/crate/cratedb-datasets/pull/9
- https://github.com/crate-workbench/cratedb-toolkit/pull/123

Reply:
I really like this way of inserting data :)

Reply:
> It is nice to remove the boilerplate code, but especially for the introductory notebooks, like `topic/timeseries/time-series-decomposition.ipynb`, we should mention how to import data via SQL code.

@ckurze: So, again, `time-series-decomposition.ipynb` did not have any database provisioning code before at all. So, we think the situation now is better than before. Please advise how you would like to improve differently, or submit a patch on a subsequent iteration.